src/Microsoft.ML.Transforms/FourierDistributionSampler.cs

```diff
@@ -117,7 +119,7 @@ void ICanSaveModel.Save(ModelSaveContext ctx)
             ctx.Writer.Write(_gamma);
         }
 
-        public float Next(Random rand)
+        float IFourierDistributionSampler.Next(Random rand)
```

Comment:
>IFourierDistributionSampler [](start = 14, length = 27)

I don't consider a regular implementation harmful in this case, since this is clearly key functionality on this object. Indeed I'm not sure this thing needs to be an interface at all. You're improving the code a bunch, but I think it still needs a bit more work. I opened #2659 to track this.

Reply:
Thank you for opening that!

Reply:
Thanks artidoro, incidentally would have been fine to leave it, but it's just as good being public.